README.md

```diff
@@ -631,6 +631,11 @@ Table of Contents
   * [unpkg.com](https://unpkg.com/) — CDN for everything on npm
   * [Web Support Revolution](https://w.tools/) — Free CDN, backup, firewall, antivirus and monitoring.
   * [Namecheap Supersonic](https://www.namecheap.com/supersonic-cdn/#free-plan) — Free DDoS protection
+  * [Arvan Cloud CDN](https://https://www.arvancloud.com/en/products/cdn)
```

Comment:
please compress to 1 line like all the other examples here, thanks!

Reply:
Will follow that.
Thanks! and Web Support Revolution{https://w.tools} no longer works